src/libsync/networkjobs.h

```diff
@@ -364,6 +364,34 @@ private slots:
     int _permanentRedirects = 0;
 };
 
+/**
+ * @brief The CheckRedirectCostFreeUrlJob class
+ * @ingroup libsync
+ */
+class OWNCLOUDSYNC_EXPORT CheckRedirectCostFreeUrlJob : public AbstractNetworkJob
+{
+    Q_OBJECT
+public:
+    explicit CheckRedirectCostFreeUrlJob(AccountPtr account, QObject *parent = nullptr);
```

Comment:
```suggestion
    explicit CheckRedirectCostFreeUrlJob(const AccountPtr &account, QObject *parent = nullptr);
```

Reply:
@mgallien but why? `AbstractNetworkJob` then takes ownership in `AccountPtr _account;` and has a nonconst value as a parameter, why change it in the child class?